packages/liveblocks-react-ui/src/primitives/AiMessage/index.tsx

```diff
@@ -98,9 +99,18 @@ function ToolInvocation({
     $types: undefined as never,
   };
   return (
-    <AiToolInvocationContext.Provider value={props}>
-      <tool.render {...props} />
-    </AiToolInvocationContext.Provider>
+    <ErrorBoundary
+      fallback={
+        <p style={{ color: "red" }}>
+          Failed to render tool call result for '{part.toolName}'. See console
+          for details.
+        </p>
+      }
+    >
```

Comment:
Does this mean that if someone uses `ToolInvocation` directly (instead of using `AiChat`), they'll see the error fallback?

Reply:
Nobody is supposed to (ever) use `ToolInvocation` directly as it's so finicky to get right. That's why we built `AiMessage.Content`. But yes, if they'd use `AiMessage.Content`, then this is also the UI they'd get. I realize this cannot be the end state. Would love some help on finalizing this in the best way possible. Both the default UI to use in `AiChat`, as well as the best default to use for non-`AiChat` use cases. We can open another PR to change that 👍